Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM node:22
 
 RUN <<EOF
-  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+  curl --silent --show-error --fail --proto "=https" --output awscliv2.zip https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip
   unzip awscliv2.zip
   rm awscliv2.zip
   ./aws/install
 
-  curl -slLo go.tar.gz https://go.dev/dl/go1.24.2.linux-amd64.tar.gz
+  curl --silent --show-error --fail --proto "=https" --location --output go.tar.gz https://go.dev/dl/go1.24.2.linux-amd64.tar.gz
```

Comment:
ADD is a handy way to doing this, because tar.gz's get auto-extracted. Really look into ADD for both of these.

Reply:
Actually, the untar operation only works for local files, not https. But the ADD option is a bit cleaner and eliminates the dependency on curl.